src/v/cluster/partition_manager.h

```diff
@@ -58,7 +59,8 @@ class partition_manager
     ntp_table_container
     get_topic_partition_table(const model::topic_namespace&) const;
 
-    inline ss::lw_shared_ptr<partition> get(const model::ntp& ntp) const {
+    inline ss::lw_shared_ptr<partition>
+    get(const model::any_ntp auto& ntp) const {
```

Comment:
One of the few places where I needed to move a method to the header since is now templated on the `any_ntp` concept.

Reply:
@dlex - yes you are right this was misplaced.